src/functions/durableEntity.ts

```diff
@@ -0,0 +1,35 @@
+import { HttpRequest, InvocationContext } from '@azure/functions';
+import * as df from 'durable-functions';
+import { DurableOrchestrationClient, IEntityFunctionContext } from 'durable-functions/lib/src/classes';
+
+df.httpClient(
+    'DurableFunctionsEntityHTTPStart',
```

Comment:
The function names should be camel case just like the file names. Also they should end in "1" to make it clear it's an instance of a function

Lastly, I don't think we need "Functions" in the name

Reply:
I picked this name just based on the existing name for durable templates

Reply:
Updated the name anyway lol